### ISSUE TITLE
Copy [EN/PT] - Onboarding Screen

### ISSUE THREAD

**Issue by OldMetalmind** [vostpt/mobile-app#44]
Determine how many screens and what text should be on each screen. 

Our suggestion is to have 4 screens with the following
- Who is VOST;
- What is this App; 
- Sources of the information for the app;
- *What kind of permissions we ask and why

Please also give us some nice images to go along.

Example of an Onboarding screen with images and text
https://www.uplabs.com/posts/onboarding-8f5bfa41-c37a-4271-84a9-1f9d905ee850

This issue is related to [Onboarding Screen #15](https://github.com/vostpt/mobile-app/issues/15)

**Comment by JorgeMiguelGomes** [comment]
@OldMetalmind I think that the part "Who is VOST" is already written?

**Comment by OldMetalmind** [comment]
@JorgeMiguelGomes I can get the text from elsewhere but the context was different, here the "Who is VOST" is more condensed, we don't have much space for text and we are in the mobile enviroment. Type of language is different. 
Just my suggestion, if not, i'll grab the text elsewhere.

**Comment by JorgeMiguelGomes** [comment]
@OldMetalmind understood loud and clear. I'll get it done today.

**Comment by JorgeMiguelGomes** [comment]
### Who Is VOST Portugal
a VOST Portugal - Associação de Voluntários Digitais em Situações de Emergência, é um grupo de cidadãos que actuam nas redes sociais com o objectivo de informar as populações com informações fidedignas 
### What is this app 
Esta aplicação pretende que tenhas o máximo de informação possível, em tempo real, das áreas onde te encontras, no que diz respeito a emergências
### Sources
Esta app usa dados de entidades oficiais como a ANEPC, IPMA, APA, DGAV, ICNF, bem como informação validada pela equipa da VOST Portugal

@OldMetalmind permissions are on your side of the court. Play ball!

**Comment by Vanethos** [comment]
@OldMetalmind I'll try to help you with that one

## Location Permission

Para que tenhas acesso às ocorrências mais perto de ti, precisamos de ter acesso á tua localização.
Caso não seja aceite a app funcionará normalmente mas não poderemos mostrar ocorrências com base na tua localização.

## Notification Permissions (for iOS)

Na app VOST podes subscrever a notificações mediante o tipo e a localização, para que estejas sempre informado das últimas ocorrências.
Para isso precisamos da tua permissão para te enviarmos notificações

@OldMetalmind and @JorgeMiguelGomes what do you think?

**Comment by JorgeMiguelGomes** [comment]
@Vanethos e @OldMetalmind I think it would be **important** to immediately tell the user that if he wants to contribute with information they will have to register an account and that all media sent via the app will be geolocated. What do you think?

**Comment by Vanethos** [comment]
@JorgeMiguelGomes if I remember correctly, but @OldMetalmind can then correct me if I'm wrong, in this version we will not have login and registration, that will be in V2

**Comment by OldMetalmind** [comment]
@Vanethos correct, This Onboarding does not include **Registration**, for this version we are not using the backend Authentication API we will need to talk with backend to see how's the status on that.

Let's discuss **Registration** offline and then create the proper issue accordingly

**Comment by JorgeMiguelGomes** [comment]
My mistake.

**Comment by Vanethos** [comment]
@JorgeMiguelGomes @OldMetalmind 

I'm going to put these into the app, however we are still missing the English Translation for these texrts

**Comment by ffrutuoso** [comment]
With suggestion from @vanvsdzn , the Notification texts should have the bold phrase changed. It is a better suited text for gender neutrality.

### Notification Permissions (for iOS)
Na app VOST podes subscrever a notificações mediante o tipo e a localização, **para que tenhas sempre informação** das últimas ocorrências.
Para isso precisamos da tua permissão para te enviarmos notificações
